### PR TITLE
add swap & update buy

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -1398,13 +1398,14 @@
             "activate": "활성화",
             "assets": "자산",
             "assetsBalance": "자산 잔액",
-            "buy": "토큰 구입",
+            "buy": "Buy",
```
Comment (vetkamak) [inline src/ko.json]: @kkkhesha could you give me a translation, pls?)

File: src/ko.json [hunk 2]
```diff
@@ -1398,13 +1398,14 @@
...
             "contacts": "연락처",
             "devers": "Devers",
             "evers": "에버",
             "invite": "초대하기",
             "receive": "받기",
             "send": "보내기",
+            "swap": "Swap",
```
Comment (vetkamak) [inline src/ko.json]: @kkkhesha and for this one as well)

File: src/ru.json [hunk 3]
```diff
@@ -1400,14 +1400,15 @@
             "invite": "Пригласить",
             "receive": "Получить",
             "send": "Отправить",
+            "swap": "Swap",
```
Comment (vetkamak) [inline src/ru.json]: @AnnStepanova could you translate it, please?)

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "buy": "구입",
```

Comment (kkkhesha) [reply src/ko.json]: @vetkamak done)

File: src/ru.json (hunk 3 above)
Comment (AnnStepanova) [reply src/ru.json]: Обменять

File: src/ko.json (hunk 2 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "swap": "바꾸기",
```

**Review by sertony — APPROVED** [review]

**Pull request merged**